### ISSUE TITLE
[UI] Leaderboard

### ISSUE THREAD

**Issue by AudreyCLEVY** [etalab-ia/eg1#63]
@dtrckd 

First Ideas:
an experience can only appear in the leaderboard if it has at least certain ‘necessary’ metrics in its evaluation set.
For Q/A evaluation : judge notator
For Rag Metric : ragas?

**Comment by dtrckd** [comment]
First question: What do we want to show in the leaderboard ? 

What I do not understand in #68 is that we do not know what we are comparing, at the difference of a given "experiment set" where it's a designed evaluation at a given time. For example, there is two "guillaume-tell" model with different score? Isn't it confusing ? And if there is a some drift (modification of model, implementation or feature), it's hard to interpret what we see.

What is your take on this ?

**Comment by AudreyCLEVY** [comment]
we want to show the best model with regard to a criterion to be defined;
pending a clearer vision, I proposed this so that the DSs could also make it their own and we could build the leaderboard together.
The notion of hyperparameters should be added to the table.
I think this is really just a first MR that will soon be enhanced.

**Comment by dtrckd** [comment]
Yes of course but we need to build it the right way in the first shot, even if it is not fully featured. Otherwise we are just adding technical debt which is the worst thing to do.  I mean we can make mistake of course,  but here is the point I try to make : 

The major issue to me, is that it seems that you are proposing a view at the experiment_set level. 
- Experiments, with no reference to their set in the platform does not make much sense to me are they are unrelated to each other.
- tracking experiment_id by hand is hard and prone to obvious error and hard to maintain from DS/user point of view.

This is the purpose of the experiment_set. So why not be experiment_set first ?

Or we remove completety the concept of experiment_set, bu I don't see the point of maintaining two concurrent thing. Maybe we need more design thinking, and with the DS, about the concepts of experiments and experiments set....

**Comment by AudreyCLEVY** [comment]
for me, if the view is at experiment_set level, then there's no logic in going to the leaderboard; you see the results in the Result tab of each experiment_set; (cf. https://github.com/etalab-ia/eg1/pull/62)

Or the leadervoard just becomes an ‘accelerated’ view of those results; and you just ‘double’ the viewing in 2 different places.
On the other hand, the current PR view has its limits in the sense that if the same combination is entered into the leaderboard several times (as a repeat, for example), then we'll only see it in the first elements of the leaderboard.

My proposal is to be able to make a Patch on the combination model X hyperparam... in order to classify it as eligible for the leaderboard... (but that would mean modifying the schema, so there's probably a simpler solution).

**Comment by AudreyCLEVY** [comment]
and I don't think the leaderboard should just be for data scientists. It is above all for the use of product managers. 
It should enable us to quickly answer the following question: which model X hyper-parameter(X collections) combination is best for conversational purposes? for summary purposes? ....
